core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4353,7 +4361,7 @@ protected RestartCause() {
     @CLIMethod(name="shutdown")
     @RequirePOST
     public void doExit( StaplerRequest req, StaplerResponse rsp ) throws IOException {
-        checkPermission(ADMINISTER);
+        checkPermission(MANAGE);
```

Comment:
can the MANAGER user recover from this?  (could they restart it).  its not a security issue theres no leak but just wondering if they should be allowed to do this (what would the use case be)